wsdiscovery/threaded.py

```diff
@@ -121,7 +121,15 @@ def _createMulticastOutSocket(self, addr, ttl):
         else:
             iface = int(addr.scope_id)
 
-        sock.setsockopt(ip_proto, self._get_multicast(), iface)
+        try:
+            sock.setsockopt(ip_proto, self._get_multicast(), iface)
+        except OSError as e:
+            logger.debug(
```

Comment:
Let's make it a warning or an error. Seems like an important thing

Reply:
done